package.json

```diff
@@ -83,18 +69,21 @@
     "karma-webpack": "2.0.3",
     "mocha": "3.2.0",
     "mockery": "2.0.0",
-    "mockgoose": "5.3.3",
     "node-sass": "4.1.1",
-    "npm-run-all": "4.0.2",
-    "react-addons-test-utils": "15.5.1",
+    "react-addons-test-utils": "15.6.2",
     "react-test-renderer": "15.5.4",
-    "sass-loader": "4.1.1",
+    "sass-loader": "^7.1.0",
     "sinon": "2.3.1",
     "sinon-chai": "2.10.0",
     "style-loader": "0.13.1",
     "supertest": "3.0.0",
-    "webpack": "2.2.1",
-    "webpack-dev-server": "2.4.1",
-    "webpack-merge": "3.0.0"
+    "webpack": "^4.16.4",
+    "webpack-cli": "^3.1.0",
```

Comment:
Typically `webpack-cli` is installed globally, isn't it? It's a developer tool, not a dev dependency